boilerplate/.eslintrc.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
       },
     ],
     "@typescript-eslint/no-var-requires": 0,
+    "@typescript-eslint/no-require-imports": 0,
+    "@typescript-eslint/no-empty-object-type": 0,
```

Comment:
Seems like the new eslint rules from Expo turned this on. I disabled it because we use this pattern quite a bit with some of the MST helpers.